vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -225,23 +250,14 @@ function Get-PrivatePackagesToCacheOnVHD {
         $dir = "c:\akse-cache\private-packages"
         New-Item -ItemType Directory $dir -Force | Out-Null
 
-        Write-Log "Downloading azcopy"
-        Invoke-WebRequest -UseBasicParsing "https://aka.ms/downloadazcopy-v10-windows" -OutFile azcopy.zip
-        Expand-Archive -Path azcopy.zip -DestinationPath ".\azcopy" -Force
-        $env:AZCOPY_AUTO_LOGIN_TYPE="MSI"
-        $env:AZCOPY_MSI_RESOURCE_STRING=$env:WindowsMSIResourceString
-
         $urls = $env:WindowsPrivatePackagesURL.Split(",")
         foreach ($url in $urls) {
             $fileName = [IO.Path]::GetFileName($url.Split("?")[0])
             $dest = [IO.Path]::Combine($dir, $fileName)
 
             Write-Log "Downloading a private package to $dest"
-            .\azcopy\*\azcopy.exe copy $URL $dest
+            Download-FileWithAzCopy -URL $URL -Dest $dest
         }
-
-        Remove-Item -Path ".\azcopy" -Force -Recurse
```

Comment:
Remove the temporary files in the same function so the work folder should not be changed.
But Download-FileWithAzCopy and Cleanup-TemporaryFiles are called in different places so it introduces an assumption that the work folder will not be changed. It will be broken if someone changes the work folder later.

Suggest creating a work folder in C:\, for exmaple, `C:\build-vhd-work-dir` and use it.

Reply:
Modified code and test again.